lib/next/test/integration/validateMessage.test.js

```diff
@@ -461,4 +308,106 @@ describe('validateMessage', () => {
       });
     });
   });
+
+  describe('... validates all properties in expected message', () => {
```

Comment:
`...`?

Reply:
Will rename, leftover.

Reply:
I was unsure how to correctly name these test cases. They seem to be a sub-suites for non-matching req/res.

Reply:
What about "always validates properties which are expected" or something along those lines?